paper/paper.bib

```diff
@@ -0,0 +1,86 @@
+@article{polian:2015,
```

Comment:
I think your bib keys are not in the correct format. https://joss.readthedocs.io/en/latest/paper.html#citations